### PR TITLE
Added bindings for system UI visibility changes.

### PR DESCRIPTION
**Pull request by skoric**
Bindings for `View.OnSystemUiVisibilityChangeListener`.

### CODE REVIEW

**Comment by JakeWharton** [comment]
Thanks. Will look tomorrow.

File: rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewSystemUiVisibilityChangeEventOnSubscribe.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            if (!subscriber.isUnsubscribed()) {
+              subscriber.onNext(ViewSystemUiVisibilityChangeEvent.create(view, visibility));
+            }
+          }
+        };
+    view.setOnSystemUiVisibilityChangeListener(listener);
+
+    subscriber.add(new MainThreadSubscription() {
+      @Override protected void onUnsubscribe() {
+        view.setOnTouchListener(null);
```
Comment (mattprecious) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewSystemUiVisibilityChangeEventOnSubscribe.java]: Did you mean `setOnSystemUiVisibilityChangeListener`?

Comment (skoric) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewSystemUiVisibilityChangeEventOnSubscribe.java]: Yes, didn't notice (autocomplete) -.- will fix it when I'm back.. thanks
for noticing

File: rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java [hunk 2]
```diff
@@ -331,6 +331,37 @@
...
+  }
+
+  /**
+   * Create an observable of system UI visibility changes for {@code view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Warning:</em> The created observable uses
+   * {@link View#setOnSystemUiVisibilityChangeListener} ()} to observe system UI visibility changes.
```
Comment (JakeWharton) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java]: Javadoc is messed up here a bit

**Comment by JakeWharton** [comment]
LGTM. Just that one Javadoc typo!

File: rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewSystemUiVisibilityChangeEventOnSubscribe.java (hunk 1 above)
Comment (JakeWharton) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewSystemUiVisibilityChangeEventOnSubscribe.java]: This is the kind of thing for which we need to be able to embed our own lint checks.

**Pull request merged**